config/routes.config.js

```diff
@@ -20,7 +20,11 @@ const routes = [
 
   // 3a/b lost income fork
   { name: 'question-gross-income', path: { en: '/gross-income', fr: '/revenu-brut' } },
+
   { name: 'question-reduced-income', path: { en: '/reduced-income', fr: '/revenu-partiel' } },
+
+  // feedback
+  { name: 'feedback', path: { en: '/feedback', fr: '/feedback' } },
```

Comment:
Needs a french route.

Reply:
The route is never visible to the user- it's just a post route ... I just switched it for a unilingual route definition (didn't know that was possible), but it still inserts a `/en/` before.